testing/simapp/upgrades.go

```diff
@@ -1,4 +1,4 @@
-package upgrades
+package simapp
```

Comment:
I personally would prefer to keep the upgrade handlers in a separate sub package.

Reply:
eh, your repo, your choice sir <3